src/rabbitstressor.js

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
So I managed to get the stressor tests running when `rabbitstressor.js` is in the `bin/` directory. However, it looks like [babel-compile](https://www.npmjs.com/package/babel-compile) only compiles files from one directory to another. I.e. it only supports one-to-one mapping. Therefore, in order to use `rabbitstressor.js`, I am importing from a `.bin/` directory (notice this is a hidden directory, just like `./test`). the require syntax looks a bit off. You can see what I mean in the latest commit.